ckanext/qdes_schema/qdes_ckan_dataset.json

```diff
@@ -145,7 +146,7 @@
       "label": "Temporal coverage start",
       "display_property": "dcat:startDate",
       "preset": "date",
-      "validators": "qdes_temporal_start_end_date",
+      "validators": "qdes_temporal_start_end_date scheming_isodatetime",
```

Comment:
This metadata field gets saved in the package_extra table as a string so needs to be converted from datetime to string using validator `convert_to_json_if_datetime`
The form is using a date picker, do we need to save this as datetime?
If not we can use the validators `isodate convert_to_json_if_date`

Reply:
no it doesn't need to be datetime, refactored to use `isodate convert_to_json_if_date`